R/dev-example.r

```diff
@@ -33,22 +33,31 @@ dev_example <- function(topic, quiet = FALSE) {
 #' @param run if `TRUE`, code in \code{\\dontrun{}} will be commented
 #'   out.
 #' @param env Environment in which code will be run.
+#' @param macros Custom macros to use to parse the `.Rd` file. See the
+#'   `macros` argument of [tools::parse_Rd()]. If `NULL`, then the
+#'   [tools::Rd2ex()] (and [tools::parse_Rd()]) default is used.
 run_example <- function(path, test = FALSE, run = FALSE,
                         env = new.env(parent = globalenv()),
-                        quiet = FALSE) {
+                        quiet = FALSE, macros = NULL) {
```

Comment:
Could we just set the default to `file.path(R.home("share"), "Rd", "macros", "system.Rd")`?

Reply:
I have no clue what the default was on the various R versions, I think it may have changed, and it could still change in the future....

Reply:
If we get rid of the 3.1 compatibility, then I can do an if-else instead. I did not do that because it would have been four cases, so do.call seemed better.

Reply:
Do you want me to do an if-else?

Reply:
Kinda :)

Reply:
OK, np.